### PR TITLE
refactor(ui): UI Navigation Refactoring

### PR DESCRIPTION
**Pull request by jjoyce0510**
**Summary**

In this PR, 2 important things are happening:

1. We are renaming the 'Manage' drop down in the Nav bar to 'Govern' - this is to more accurately represent its contents going forward.
2. We are moving _both_ Access Policy management and Users & Groups management into the Settings panel. Don't worry - we redirect from the previous urls to the new settings locations (your urls will continue to work). This is to clear space and declutter the navigation bar, which we know is overflowing for smaller screens. 

We are super excited about these changes, they will clear the runway for more functionality and improve the organization of the app significantly. See the changes in the screenshots below.

**Changes**

1. Remove users & groups, policies from top nav bar. 
2. Add users & groups, policies inside an "Access" section of the settings panel. only visible if the correct platform privileges are maintained. 
4. Add copy URL action buttons to Container, Domain, Tag pages. 
5. Other minor styling improvements.

**Screenshots**
Screen Shot 2022-06-02 at 4 44 54 PM

Screen Shot 2022-06-02 at 4 44 44 PM

Screen Shot 2022-06-02 at 4 44 39 PM

Screen Shot 2022-06-02 at 4 44 34 PM

Screen Shot 2022-06-02 at 10 33 55 AM
Screen Shot 2022-06-02 at 10 34 13 AM

## Checklist
- [ ] The PR conforms to DataHub's [Contributing Guideline](https://github.com/datahub-project/datahub/blob/master/docs/CONTRIBUTING.md) (particularly [Commit Message Format](https://github.com/datahub-project/datahub/blob/master/docs/CONTRIBUTING.md#commit-message-format))
- [ ] Links to related issues (if applicable)
- [ ] Tests for the changes have been added/updated (if applicable)
- [ ] Docs related to the changes have been added/updated (if applicable). If a new feature has been added a Usage Guide has been added for the same.
- [ ] For any breaking change/potential downtime/deprecation/big changes an entry has been made in [Updating DataHub](https://github.com/datahub-project/datahub/blob/master/docs/how/updating-datahub.md)

### CODE REVIEW

File: datahub-web-react/src/app/entity/shared/containers/profile/header/EntityHeader.tsx [hunk 1]
```diff
@@ -189,15 +190,7 @@ export const EntityHeader = ({ refreshBrowser, headerDropdownItems, isNameEditab
...
-                    <Tooltip title="Copy URN. An URN uniquely identifies an entity on DataHub.">
-                        <Button
-                            icon={copiedUrn ? <CheckOutlined /> : <CopyOutlined />}
-                            onClick={() => {
-                                navigator.clipboard.writeText(urn);
-                                setCopiedUrn(true);
-                            }}
-                        />
-                    </Tooltip>
+                    <CopyUrn urn={urn} isActive={copiedUrn} onClick={() => setCopiedUrn(true)} />
```
Comment (chriscollins3456) [inline datahub-web-react/src/app/entity/shared/containers/profile/header/EntityHeader.tsx:193]: love it

**Review by chriscollins3456 — APPROVED** [review]
Looks good!

**Review by dexter-mh-lee — APPROVED** [review]
Looks amazing!!!

File: datahub-web-react/src/app/settings/SettingsPage.tsx [hunk 2]
```diff
@@ -28,12 +34,48 @@ const PageTitle = styled(Typography.Title)`
...
+`;
+
+const ItemTitle = styled.span`
+    margin-left: 8px;
+`;
+
+/**
+ * URL Paths for each settings page.
+ */
+const PATHS = [
```
Comment (chriscollins3456) [inline datahub-web-react/src/app/settings/SettingsPage.tsx:49]: maybe `SETTINGS_PATHS`? was confused below when you were mapping over "PATHS"

File: datahub-web-react/src/app/settings/SettingsPage.tsx [hunk 3]
```diff
@@ -43,19 +85,48 @@ export const SettingsPage = () => {
...
+                        )}
                     </Menu>
                 </SettingsBarContainer>
-                {selectedKey === 'access-tokens' && <AccessTokens />}
+                <Switch>
+                    <Route exact path={path}>
+                        <Redirect to={`${pathname}${pathname.endsWith('/') ? '' : '/'}${DEFAULT_PATH.path}`} />
+                    </Route>
+                    {PATHS.map((p) => (
+                        <Route path={`${path}/${p.path.replace('/', '')}`} render={() => p.content} key={p.path} />
```
Comment (chriscollins3456) [inline datahub-web-react/src/app/settings/SettingsPage.tsx:127]: why do you have this `replace` logic here if there's no "/" in any of the paths above?

**Review by chriscollins3456 — APPROVED** [review]
Looks good! two small nits but it looks like this PR was merged while I was re-reviewing

**Comment by mydq** [comment]
When will this pr be merged and released?

**Pull request merged**